wordpress-mobile/WordPress-iOS
# Fix editor More button display on iOS 14b2

Fixes #14460.

There's an issue on iOS 14b2 where the contents of the editor More menu isn't visible when displayed as a popover on iPad. I discovered that this is fixed if we display the popover from a `sourceRect` instead of a `barButton`. This seems like an OS-level issue as I can't see we're doing anything wrong, so hopefully we can remove this change in a future PR.

Note that this issue affects both Aztec and Gutenberg.

![editor-popover-1](https://user-images.githubusercontent.com/4780/87241381-84893600-c41a-11ea-947b-f65a05d6fb91.png)

**To test:**

* You'll need to build using Xcode 11.5 to an iPad running iOS 14.2. To do this, you'll also need the latest Xcode 12 beta installed, and to symlink to the device support folder as detailed here: https://gist.github.com/steipete/d9b44d8e9f341e81414e86d7ff8fb62d#file-gistfile1-txt
* Build and run on an iPad running iOS 14b2
* Open Aztec and open the More menu multiple times. Ensure you can see the contents.
* Open the block editor and open the More menu multiple times. Ensure you can see the contents.

PR submission checklist:

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary. (I don't think we should or can mention iOS 14 changes in release notes).
